### ISSUE TITLE
Xdebug path doesn't exist on installation?

### ISSUE THREAD

**Issue by Schrank** [puphpet/puphpet#2020]
The error returned is:

```
==> default: Error: cp /.puphpet-stuff/xdebug/modules/xdebug.so /usr/lib/php/20151012/xdebug.so returned 1 instead of one of [0]
==> default: Error: /Stage[main]/Puphpet::Php::Xdebug/Exec[cp /.puphpet-stuff/xdebug/modules/xdebug.so /usr/lib/php/20151012/xdebug.so]/returns: change from notrun to 0 failed: cp /.puphpet-stuff/xdebug/modules/xdebug.so /usr/lib/php/20151012/xdebug.so returned 1 instead of one of [0]

````

I assume, this happens, because this fails:

```puppet
puphpet/puppet/modules/puphpet/manifests/php/xdebug.pp:29
  } elsif $puphpet::php::settings::enable_xdebug {
    # php 5.6 requires xdebug be compiled, for now
    case $::operatingsystem {
      # Debian and Ubuntu slightly differ
      'debian', 'ubuntu': {
        if is_dir('/usr/lib/php5/20131226-zts') {
          $mod_dir = '/usr/lib/php5/20131226-zts'
        } elsif is_dir('/usr/lib/php5/20131226') {
          $mod_dir = '/usr/lib/php5/20131226'
        } else {
          $mod_dir = '/usr/lib/php/20151012'
        }
      }
      'redhat', 'centos': {$mod_dir = '/usr/lib64/php/modules'}
    }
```
Before the update it looked like this, so the last elsif was an else and the `zts` directory is missing. <b>With the old version it is working.</b>

````
  } elsif $puphpet::php::settings::enable_xdebug {
    # php 5.6 requires xdebug be compiled, for now
    case $::operatingsystem {
      # Debian and Ubuntu slightly differ
      'debian', 'ubuntu': {
        if is_dir('/usr/lib/php5/20131226-zts') {
          $mod_dir = '/usr/lib/php5/20131226-zts'
        } else {
          $mod_dir = '/usr/lib/php5/20131226'
        }
      }
      'redhat', 'centos': {$mod_dir = '/usr/lib64/php/modules'}
    }
```



```yaml

vagrantfile:
    target: local
    vm:
        box: puphpet/ubuntu1404-x64
        box_url: puphpet/ubuntu1404-x64
        hostname: example.de.dev
        memory: '1024'
        cpus: '2'
        chosen_provider: parallels
        network:
            private_network: 192.168.58.152
            forwarded_port:
                vflnp_8bef3lp9kqo6:
                    host: '5731'
                    guest: '22'
        post_up_message: ''
        provider:
            virtualbox:
                modifyvm:
                    natdnshostresolver1: 'on'
                showgui: '0'
            vmware:
                numvcpus: 1
            parallels:
                cpus: 1
        provision:
            puppet:
                manifests_path: puphpet/puppet/manifests
                module_path: puphpet/puppet/modules
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
        synced_folder:
            vflsf_ov1u0ii6y9go:
                source: ./
                target: /var/www
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
        usable_port_range:
            start: 10200
            stop: 10500
    ssh:
        host: null
        port: null
        private_key_path: null
        username: vagrant
        guest_port: null
        keep_alive: true
        forward_agent: false
        forward_x11: false
        shell: 'bash -l'
        insert_key: false
    vagrant:
        host: detect
server:
    install: '1'
    packages:
        - vim
users_groups:
    install: '1'
    groups: {  }
    users: {  }
locale:
    install: '1'
    settings:
        default_locale: ''
        locales:
            - en_GB.UTF-8
            - en_US.UTF-8
firewall:
    install: '1'
    rules: {  }
cron:
    install: '1'
    jobs: {  }
nginx:
    install: '0'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
        proxy_connect_timeout: 600s
        proxy_send_timeout: 600s
        proxy_read_timeout: 600s
        names_hash_bucket_size: 128
    upstreams: {  }
    vhosts:
        nxv_4fcttvo4kfe6:
            server_name: example.dev
            server_aliases:
                - www.example.dev
            www_root: /var/www/
            listen_port: '80'
            index_files:
                - index.php
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: ''
            ssl_protocols: ''
            ssl_ciphers: ''
    proxies: {  }
apache:
    install: '1'
    settings:
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - headers
        - proxy_fcgi
        - rewrite
    vhosts:
        av_l5ipusgy2k7c:
            servername: example.dev
            serveraliases:
                - www.example.dev
            docroot: /var/www
            port: '80'
            setenv:
                - 'MAGE_IS_DEVELOPER_MODE true'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_cnes1zkfq0dz:
                    path: /var/www
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_5sinz0vg80u8:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
php:
    install: '1'
    settings:
        version: '56'
    modules:
        php:
            - cli
            - intl
            - curl
            - gd
            - mcrypt
        pear: {  }
        pecl: {}

    ini:
        display_errors: 'On'
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        date.timezone: UTC
    fpm_ini:
        error_log: /var/log/php-fpm.log
    fpm_pools:
        phpfp_699eke6fhh23:
            ini:
                prefix: www
                listen: '127.0.0.1:9000'
                security.limit_extensions: .php
                user: www-user
                group: www-data
    composer: 1
    composer_home: ''
xdebug:
    install: '1'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_connect_back: '1'
        xdebug.remote_enable: '1'
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
blackfire:
    install: '0'
    settings:
        server_id: ''
        server_token: ''
        agent:
            http_proxy: ''
            https_proxy: ''
            log_file: stderr
            log_level: '1'
        php:
            agent_timeout: '0.25'
            log_file: ''
            log_level: '1'
xhprof:
    install: '0'
wpcli:
    install: '0'
    version: ''
drush:
    install: '0'
    version: ''
ruby:
    install: '1'
    versions: {  }
python:
    install: '1'
    packages: {  }
    versions: {  }
nodejs:
    install: '0'
    npm_packages: {  }
hhvm:
    install: '0'
    nightly: 0
    composer: '1'
    composer_home: ''
    settings: {  }
    server_ini: {  }
    php_ini: {  }
mariadb:
    install: '0'
    settings:
        version: '10.0'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users: {  }
    databases: {  }
    grants: {  }
mysql:
    install: '1'
    settings:
        version: '5.5'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users:
        mysqlnu_4kz028smjrx5:
            name: example_de
            password: 793BHnxYfX2PZMS5
    databases:
        mysqlnd_dzddmzkk8fes:
            name: example_de
            sql: /var/www/doc/example.sql
    grants:
        mysqlng_3t14gml8zdp0:
            user: example_de
            table: 'example_de.*'
            privileges:
                - ALL
postgresql:
    install: '0'
    settings:
        global:
            encoding: UTF8
            version: '9.3'
        server:
            postgres_password: ''
    databases: {  }
    users: {  }
    grants: {  }
    adminer: 0
mongodb:
    install: '0'
    settings:
        auth: 1
        bind_ip: 127.0.0.1
        port: '27017'
    databases: {  }
redis:
    install: '0'
    settings:
        conf_port: '6379'
sqlite:
    install: '0'
    adminer: 0
    databases: {  }
mailhog:
    install: '1'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '1080'
        path: /usr/local/bin/mailhog
beanstalkd:
    install: '0'
    settings:
        listenaddress: 0.0.0.0
        listenport: '13000'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: 0
rabbitmq:
    install: '0'
    settings:
        port: '5672'
    users: {  }
    vhosts: {  }
    plugins: {  }
elastic_search:
    install: '0'
    settings:
        version: null
        java_install: true
solr:
    install: '0'
    settings:
        version: 4.10.2
        port: ''

**Comment by simonihmig** [comment]
Same here

**Comment by jtreminio** [comment]
Fixed by 1ab6605628ebc253200686d86e4fd1974cbe15eb
